docs/BR.md

```diff
@@ -380,6 +382,10 @@ The Definitions found in the CA/Browser Forum's Network and Certificate System S
 
 **Legal Entity**: An association, corporation, partnership, proprietorship, trust, government entity or other entity with legal standing in a country's legal system.
 
+**Multi-Perspective Issuance Corroboration**: A process by which the determinations made during domain validation and CAA checking by the Primary Network Perspective are corroborated from other Network Perspectives before Certificate issuance. 
```

Comment:
```suggestion
**Multi-Perspective Issuance Corroboration**: A process by which the determinations made during domain validation and CAA checking by the Primary Network Perspective are corroborated by other Network Perspectives before Certificate issuance. 
```

Reply:
We have no issues using “by" in place of “from.” The original use of “from" was intended to describe the location *from* which MPIC lookups take place, but if this is clearer for the community - we’re happy to use it. 

We pre-staged this change in https://github.com/ChristopherRC/servercert/commit/07f9c3bac410a8af53ef60755b2788e67f7ab556 in preparation for a subsequent round of discussion.